addOns/retire/src/main/java/org/zaproxy/addon/retire/RetireScanRule.java

```diff
@@ -61,16 +60,15 @@ public int getPluginId() {
 
     @Override
     public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
-        if (!getHelper().isPage200(msg) || getRepo() == null) {
-            return;
-        }
-        String uri = msg.getRequestHeader().getURI().toString();
-        if (!ResourceIdentificationUtils.isImage(msg) && !ResourceIdentificationUtils.isCss(msg)) {
-            Repo scanRepo = getRepo();
+        Repo scanRepo = getRepo();
```

Comment:
This is now initialising even if not needed (that's why it was in the right branch), not that it matters much, should be sooner or later.